TypeScript documentation for _app.tsx (#10345)
zeit/next.js: docs/basic-features/typescript.md

  res.status(200).json({ name: 'John Doe' })
}
```

> ## Custom `App` using `_app.tsx`

Reviewer: I'll prefer either a matching title with the other section, like the other titles in this section, or to be even more specific with:
```
## Custom `App` using `pages/_app.tsx
```

```suggestion
## Custom `App`
```

> If you’d like to [customize the `App` component](/docs/advanced-features/custom-app) in TypeScript, create `pages/_app.tsx`:

Reviewer: Alternative with the wording that's being used by the rest of this section:

```suggestion
If you have a [custom `App` ](/docs/advanced-features/custom-app), you can use the built-in type `AppProps`, like so:
```
